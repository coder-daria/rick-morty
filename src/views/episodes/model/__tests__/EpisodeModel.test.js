import {
  EPISODE_DETAILS_MOCKED_DATA,
  EPISODE_FORMATED_MOCKED_DETAILS,
  EPISODE_NOT_FORMATED_MOCKED_DETAILS,
} from '../../__mocks__/episode-mock-data';

import getEpisodeData from '../EpisodeModel';

describe('EpisodeModel', () => {
  it('should return episode data', () => {
    const result = getEpisodeData({
      ...EPISODE_DETAILS_MOCKED_DATA,
      ...EPISODE_NOT_FORMATED_MOCKED_DETAILS,
    });

    expect(result).toMatchSnapshot({
      ...EPISODE_DETAILS_MOCKED_DATA,
      ...EPISODE_FORMATED_MOCKED_DETAILS,
    });
  });
});
