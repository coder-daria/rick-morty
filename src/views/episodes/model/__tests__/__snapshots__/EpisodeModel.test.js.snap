// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodeModel should return episode data 1`] = `
Object {
  "air date": "11/25/2020, 4:59:08 PM",
  "characters": Array [
    Object {
      "name": "Tom",
    },
  ],
  "created": "11/25/2019, 4:59:08 PM",
  "episode": "0E12",
  "name": "Sunny day",
}
`;
